source/extensions/omni.isaac.lab/omni/isaac/lab/managers/reward_manager.py

```diff
@@ -107,7 +107,7 @@ def reset(self, env_ids: Sequence[int] | None = None) -> dict[str, torch.Tensor]
             # store information
             # r_1 + r_2 + ... + r_n
             episodic_sum_avg = torch.mean(self._episode_sums[key][env_ids])
-            extras["Episode Reward/" + key] = episodic_sum_avg / self._env.max_episode_length_s
+            extras["Episode_Reward/" + key] = episodic_sum_avg / self._env.max_episode_length_s
```

Comment:
Why do we even have "Episode" as a prefix. Shall we just call it Reward? Also this is breaking again since all the prior logs will appear differently :(

Reply:
This is somehow hydra related, I get all of a sudden some weird tensorboard warnings that "Episode Reward" is not a valid string.

Reply:
Huh.... Why does hydra care about tensorboard :/ . I won't have time this Thursday to check it out but I feel this shouldn't be affected by Hydra o.O

Reply:
yes this is super strange